### PR TITLE
connect base_scraper to persistence layer

### PR DESCRIPTION
**Pull request by terinpw**

### CODE REVIEW

**Comment by jessex** [comment]
Now that we are officially no longer writing to Datastore from descendants of the `BaseScraper`, can we update the README to note how to verify data that is being ingested locally?

File: recidiviz/persistence/persistence.py [hunk 1]
```diff
@@ -20,16 +20,23 @@
...
-    surname/birthday already exists, then update that person.
+    If in prod, persist each person in the ingest_info. If a person with the
+    given surname/birthday already exists, then update that person.
+
+    If not in prod, simply print the given ingest_infos for debugging
 
     Args:
          ingest_info: The IngestInfo containing each person
     """
+    if not environment.in_prod():
```
Comment (jessex) [inline recidiviz/persistence/persistence.py]: Would you mind making this toggle-able via an environment variable or flag of some sort? Some of us would be able to persist data locally without issue.

File: recidiviz/persistence/persistence.py [hunk 2]
```diff
@@ -20,16 +20,23 @@
...
+        print ingest_info
```
Comment (jessex) [inline recidiviz/persistence/persistence.py]: I think we should change this to an actual logging call. Even for local usage, there are some [advantages](https://www.loggly.com/blog/4-reasons-a-python-logging-library-is-much-better-than-putting-print-statements-everywhere/).

File: recidiviz/utils/environment.py [hunk 3]
```diff
@@ -46,6 +46,7 @@ def in_prod():
         False if not
     """
     current_environment = os.getenv('SERVER_SOFTWARE', '')
+    print "TERINPW: current_environ = " + str(current_environment)
```
Comment (jessex) [inline recidiviz/utils/environment.py]: Remember sure to drop this or change it to a log statement before merging!

File: recidiviz/persistence/persistence.py (hunk 1 above)
Comment (terinpw) [reply recidiviz/persistence/persistence.py]: Yup that makes sense, added it to the environment variables.

File: recidiviz/persistence/persistence.py (hunk 2 above)
Comment (terinpw) [reply recidiviz/persistence/persistence.py]: Good point, done.

File: recidiviz/utils/environment.py (hunk 3 above)
Comment (terinpw) [reply recidiviz/utils/environment.py]: Yup, realized that was there and removed!

**Pull request merged**